test/trace/integrations/rails_integration_test.rb

```diff
@@ -0,0 +1,129 @@
+# Copyright 2017 OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "fileutils"
+require "open3"
+require "timeout"
+require "test_helper"
+require "opencensus/trace/integrations/rails"
+require "faraday"
+
+describe "Rails integration" do
+  module RailsTestHelper
+    APP_NAME = "railstest"
+    BASE_DIR = File.absolute_path File.dirname File.dirname File.dirname __dir__
+    TMP_DIR = File.join BASE_DIR, "tmp"
```

Comment:
Could avoid having the tmp dir generated in the source tree with https://ruby-doc.org/stdlib-2.2.0/libdoc/tmpdir/rdoc/Dir.html#method-c-mktmpdir 

It could be nice to see the resulting rails app for testing so maybe this is fine.

Reply:
Yeah that's why it's currently being generated in the dir tree, so I can look at the generated app and figure out why the test is failing...